SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialLoadBalanceEnforcer.h

```diff
@@ -3,52 +3,71 @@
 #pragma once
 
 #include "Interop/SpatialStaticComponentView.h"
+#include "Schema/AuthorityIntent.h"
+#include "Schema/ComponentPresence.h"
+#include "Schema/NetOwningClientWorker.h"
+#include "Schema/StandardLibrary.h"
 #include "SpatialCommonTypes.h"
 
-#include <WorkerSDK/improbable/c_worker.h>
-
-#include "CoreMinimal.h"
+#include "SpatialView/EntityComponentTypes.h"
+#include "SpatialView/EntityDelta.h"
+#include "SpatialView/EntityView.h"
+#include "SpatialView/SubView.h"
```

Comment:
Guess we could forward declare Subview?

Reply:
Yeah there's a ton of includes here, forward declare where possible.

Reply:
Tried it, ended up with a bunch of issues around it being an incomplete type after including in the cpp. Is there something I'm missing?

Reply:
I think the only one you could forward declare is FSubView although it seems unlikely that something is going to include this and not include that anyway.
IMO these are all pretty small includes and there's not that many but that can be forward declared and included in the cpp file.

Reply:
I think the forward declaration of subview works now, although it seemed to be hiding the dependency for EntityACL, so I am including standard library again